### PR TITLE
VS Code Preview first draft #68

### PR DESCRIPTION
**Pull request by irinaponomarevahaulmont**
Please review the first draft.

### CODE REVIEW

File: content/modules/vscode-extension/pages/vscode-preview.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+The Amplicode extension provides functionality that allows th user to interactively preview application's components directly from the IDE.
+
+[[using-preview]]
+== Using Preview
+
+If the Amplicode extension is installed, the .tsx files, which contain the source code for components, will also contain preview hints, see example below.
+
+image::preview/hint.png[align=center, width=700]
+
+The file used in this example, `ToggleThemeButton.txs`, is located under `src` -> `app` -> `app-bar` and is present in any frontend project generated with the help of the Amplicode extension. It is responsible for the button that switches xref:vscode-extension:vscode-themes.adoc[themes] between the light and the dark mode.
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/vscode-preview.adoc:12]: .txs --> .tsx

File: content/modules/vscode-extension/pages/vscode-preview.adoc [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+Once we click on any hint, e.g. the *_Open ToggleThemeButton preview_* hint, the build process starts. It will be emulating the dev mode to make it possible to open the component preview without opening the entire frontend UI. If the process is successful, a separate window will open in a new tab, and the component will be shown in it.
+
+image::preview/preview-example.png[align=center, width=700]
+
+The component in the preview mode is interactive and toggles. If you click on it, its look will change in just the same way as it does when the complete frontend is opened in the browser.
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/vscode-preview.adoc:18]: > and toggles

strange. you can delete it, you've described the behavior in th next sentence.

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Comment by irinaponomarevahaulmont** [comment]
Please review the changes

File: content/modules/vscode-extension/pages/vscode-preview.adoc [hunk 3]
```diff
@@ -0,0 +1,32 @@
+= VS Code Preview
```
Comment (alexbudarov) [inline content/modules/vscode-extension/pages/vscode-preview.adoc:1]: VS Code Preview --> Preview

**Review by alexbudarov — CHANGES_REQUESTED** [review]

**Comment by irinaponomarevahaulmont** [comment]
Please review the second round.

**Pull request merged**